<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontReportQueryTable</class>
 <widget class="QWidget" name="FrontReportQueryTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>512</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QTableView" name="tableViewQuery"/>
    </item>
    <item>
     <widget class="QTableView" name="tableViewReport"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuery">
      <property name="text">
       <string>查詢</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExport">
      <property name="text">
       <string>匯出(.CSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
